### DISCUSSION TITLE
[iOS, iPhone 13] Performance question

### DISCUSSION THREAD

**Discussion by deWaardt** [utmapp/UTM#4763] [UTM Help (iOS)]
Good morning people,

I'm purely messing around here and not really trying to make anything usable.. But I've been questioning what kind of performance I should expect.

I'm using a non-jailbroken iPhone 13 using the tethered method. I can't even get a remotely usable experience. I was able to install Debian, Fedora and Windows 11 ARM edition without too much hassle, but it's EXTREMELY slow.

Like, not regular slow but much slower than I think it could be. Opening the start menu on Windows 11 takes 5 minutes and opening settings on Fedora takes two to three minutes. People seem to have having faster boot times emulating an x86 Windows 10 machine.

I've seen many other people on YouTube run Linux or even Win11 on their iPhone and it was much snappier. I can't even install SPICE tools because opening an explorer window already takes 10 minutes.

I am forcing multicore, tried different display adapters and different CPU models, but nothing really changes.

Is this just how it is, or am I doing something completely wrong?

**Comment by stossy11** [comment]
I have the same thing so idk

I’m on an iPad Air 5 M1

**Comment by MalxWare47** [comment]
LMAO WINDOWS 11 on the iphone. xD yeah i highly doubt that. why would u even want to use windows on an iphone. i have the 12 and the best i got was windows 7 running smoooth but with jit. theres no way u could possibly get windows 11 maybe with m2

**Reply by steve500** [reply]
Windows 11 arm installer ISO does boot up to the PE environment for installation on my iphone 15 pro. So, possible? Sure!
